mackup/applications/sublime-text-3.cfg

```diff
@@ -2,6 +2,7 @@
 name = Sublime Text 3
 
 [configuration_files]
-Library/Application Support/Sublime Text 3/Packages/User
```

Comment:
According to this: https://packagecontrol.io/docs/syncing, we should sync the Packages/User folder and not Packages and Installed Packages. I think the earlier configuration was correct.